utils/numbers.ts

```diff
@@ -29,3 +29,11 @@ export const formatNumber = (value: number, decimals = 2): string => {
 
   return `${value.toFixed(decimals)}`;
 };
+
+export const setKeyfactsDecimalCount = (flag: string | undefined): number => {
```

Comment:
Is this something that'll only ever benefit the key facts section? If it's only for the key facts, then remove it from the general utils folder.

Reply:
Yes seems its only Keyfacts that has an exception for decimal places in the rank value. Have moved this into `IndicatorStat` utils